rollup/src/removeDynamic.ts

```diff
@@ -50,11 +50,11 @@ function isNullOrWhitespace(value:string) {
 export default function dynamicRemove(options:IDynamicProtoRollupOptions = {}) {
   var token = (options || {}).tagname || "@DynamicProtoStub";
   var replaceValue = (options || {}).comment || "// Removed Stub for %function%.";
-  let tokenGroups:Array<number> = [4, 9, 12];
+  let tokenGroups:Array<number> = [4, 10, 13];
```

Comment:
This is the fixes for the rollup dynamic remove to be able to detect and remove methods that are defined to use "..." and default values.